framework/wazuh/tests/test_vulnerability.py

```diff
@@ -29,11 +30,12 @@
 test_data_cve = InitAgent(data_path=test_data_path, db_name='schema_cve_test.sql')
 
 
-def send_msg_to_wdb(msg, raw=False, *args, **kwargs):
-    # Tests for CVE endpoint use a different schema and query
+def send_msg_to_wdb(msg, raw=False):
     query = msg[msg.find('sql') + len('sql '):]
-    result = test_data.cur.execute(query).fetchall() if msg.startswith('global') else test_data_cve.cur.execute(query).fetchall()
-    return list(map(remove_nones_to_dict, map(dict, result)))
+    result = test_data.cur.execute(query).fetchall() if msg.startswith('global') else test_data_cve.cur.execute(
+        query).fetchall()
+    result = list(map(remove_nones_to_dict, map(dict, result)))
+    return ['ok', dumps(result)] if raw else result
```

Comment:
Consider adding this function to test utils instead of declaring it in every test module.

Reply:
I think it is not possible since `test_data` is an instance of an object created in each test module. Using `getattr` was unsuccessful too.